content/uk/gallery1/index.md

```diff
@@ -0,0 +1,66 @@
+---
+author: "Hugo Authors"
+title: "4-6 listopada. Vistavka"
+type: gallery
+---
+
+<div class='text-left mt-8 mx-auto container px-6 text-justify mb-20'>
+    <h2 class='uppercase text-5xl text-red-600 font-bold py-4 mb-4'>
+        Перегляньте фото <br> з наших заходів
+    </h2>
+    <p class='text-2xl text-gray-600 dark:text-white'>
+        Якщо Ви хочете використати наше фото, будь ласка вказуйте <br> автора, або посилання на сайт.
+    </p>
+</div>
+
+
+<div class='text-left mt-8 mx-auto container px-6 text-justify flex flex-col'>
+<div class = 'flex flex-row items-baseline'>
+<a href = '/uk/albums/'>
+<svg viewBox="0 0 24 24" fill="currentColor" width="24" height="24" class="s2X7Ms" aria-hidden="true"><path fill="currentColor" d="M14.5 4.92893219L15.2071068 5.63603897 8.84243864 11.9992929 15.2071068 18.363961 14.5 19.0710678 7.42893219 12z" fill-rule="evenodd"></path></svg>
+</a>
+    <h3 class="text-4xl font-bold mb-20 ml-4 flex-1 grow">
+        4-6 листопада. Виставка
+    </h3>
+    <div class='flex-none justify-center align-sub font-bold text-gray-900 pt-4 dark:text-white'>
```

Comment:
Let's comment sorting and adding files for now because for downloading in such a way we need a backend and we don't have one.

Maybe later we will come up with something.